pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.github.FRC125</groupId>
+    <artifactId>maven-build-files<</artifactId>
+    <version>1.0.5</version>
+  </parent>
+
+  <artifactId>FRamework</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <dependencies>
```

Comment:
All versions are specified in the parent pom so that you can ensure that everyone uses the same version of something across multiple projects.